src/admin/pages/posts/post.component.ts

```diff
@@ -35,10 +35,22 @@ export class PostComponent implements OnInit, OnDestroy {
 
     // File Upload
     private _imageToUpload = null;
+    private _imagePreviewContainer: HTMLElement;
 
     setFile({ target: { files } } = <any>{}) {
+        let imgPreviewContainer = this._imagePreviewContainer = this._document.getElementById('imagePreview');
         let [file] = files;
         this._imageToUpload = file;
+
+        let reader = new FileReader();
```

Comment:
Since this is an internal application I don't think it's necessary. You are welcome to add the bits and make a commit.